commit dfae3ffa0f5290d2ee1dcadbd6c125de468f032f
Author: Ian Romanick <ian.d.romanick@intel.com>
Date:   Tue Mar 1 13:35:39 2011 -0800

    intel: Remove driver date and related bits from renderer string
    
    Not only did this contain lies, it contained lies that wouldn't be
    useful even if true.
    (cherry picked from commit b0a7492aebeb4517346f0da2362d6991a7385b59)

diff --git a/src/mesa/drivers/dri/intel/intel_context.c b/src/mesa/drivers/dri/intel/intel_context.c
index 00c51a0..d5545c9 100644
--- a/src/mesa/drivers/dri/intel/intel_context.c
+++ b/src/mesa/drivers/dri/intel/intel_context.c
@@ -62,10 +62,6 @@ int INTEL_DEBUG = (0);
 #endif
 
 
-#define DRIVER_DATE                     "20100330 DEVELOPMENT"
-#define DRIVER_DATE_GEM                 "GEM " DRIVER_DATE
-
-
 static const GLubyte *
 intelGetString(GLcontext * ctx, GLenum name)
 {
@@ -182,7 +178,7 @@ intelGetString(GLcontext * ctx, GLenum name)
          break;
       }
 
-      (void) driGetRendererString(buffer, chipset, DRIVER_DATE_GEM, 0);
+      (void) driGetRendererString(buffer, chipset, "", 0);
       return (GLubyte *) buffer;
 
    default:
